projects/addon-mobile/directives/ripple/ripple.style.less

```diff
@@ -5,24 +5,21 @@
     }
 
     to {
-        transform: none;
         opacity: 0.12;
     }
 }
 
 @keyframes tuiRippleOff {
     from {
-        transform: none;
         opacity: 0.12;
     }
 
     to {
-        transform: none;
         opacity: 0;
     }
 }
 
-*[tuiRipple] {
+.tui-ripple-container {
```

Comment:
Maybe we shouldn't do that and leave it as the responsibility of the dev providing the selectors? What if they have `position: absolute` or `overflow: clip` and it would come into conflict? What do you think?